src/globus_sdk/experimental/globus_app/_validating_token_storage.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         :param consent_client: An AuthClient to be used for consent polling. If omitted,
             dependent scope requirements are ignored during validation.
         """
-        self._token_storage = token_storage
+        self.token_storage = token_storage
```

Comment:
Potentially worth calling this out in the changelog as well. Wasn't sure whether that rose to that level